ios/RNFirebase/notifications/RNFirebaseNotifications.m

```diff
@@ -214,9 +223,16 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
          withCompletionHandler:(void(^)())completionHandler NS_AVAILABLE_IOS(10_0) {
 #endif
      NSDictionary *message = [self parseUNNotificationResponse:response];
+           
+     NSString *handlerKey = message[@"notification"][@"notificationId"];
 
      [self sendJSEvent:self name:NOTIFICATIONS_NOTIFICATION_OPENED body:message];
-     completionHandler();
+     if (handlerKey != nil) {
+         completionHandlers[handlerKey] = completionHandler;
+     }
+     else {
```

Comment:
Could you move this to the same line - `} else {` thanks

Reply:
will fix